src/marten/handlers/base.cr

```diff
@@ -125,24 +126,36 @@ module Marten
       end
 
       # Returns an empty response associated with a given status code.
-      def head(status : Int32) : HTTP::Response
-        HTTP::Response.new(content: "", content_type: "", status: status)
+      def head(status : StatusCode) : HTTP::Response
+        HTTP::Response.new(
+          content: "",
+          content_type: "",
+          status: HTTP::Status.status_code(status)
+        )
       end
```

Comment:
Could we leverage method overloading in order to avoid the extra call to the `HTTP::Status#status_code` method when providing an int?

For example:

```crystal
# Returns an empty response associated with a given status code.
def head(status : Int32) : HTTP::Response
  HTTP::Response.new(content: "", content_type: "", status: status)
end

# :ditto:
def head(status : Symbol) : HTTP::Response
  head(status: HTTP::Status.status_code(status))
end
```